build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Build waypoint image in the x86 env:
+# ARCH=x86 bash build.sh
+#
+# Build waypoint image in the arm env:
+# ARCH=arm bash build.sh
+
+REPO=${1:-"kmesh-net"}
+
+# Build Envoy
+BUILD_WITH_CONTAINER=1 make build_envoy
+
+docker create --name temp -v cache:/home/.cache busybox
```

Comment:
what does this container do?

Reply:
And it is not deleted

Reply:
> what does this container do?

The envoy binary is stored in the `cache` volume, and the `temp` container is used to copy the envoy binary from the volume.

Reply:
should we remove the cache volume later

Reply:
> should we remove the cache volume later

I think it's not necessary. If you build in a CI env, it doesn't matter whether you delete it or not.

If you build in a local env, the cache can greatly speed up your build time. Next time, you only need to compile the modified files. Otherwise, compiling envoy from scratch every time will be very slow.

Reply:
make sense